src/main/daml/Daml/Finance/Interface/Holding/V4/Factory.daml

```diff
@@ -5,11 +5,11 @@
 
 module Daml.Finance.Interface.Holding.V4.Factory where
 
-import Daml.Finance.Interface.Holding.V4.Holding qualified as Holding (I)
 import Daml.Finance.Interface.Types.Common.V3.Types (AccountKey(..), HoldingFactoryKey(..), Id, InstrumentKey(..), Parties, PartiesMap)
 import Daml.Finance.Interface.Util.V3.Disclosure qualified as Disclosure (AddObservers(..), I, RemoveObservers(..), flattenObservers)
 import Daml.Finance.Interface.Util.V3.InterfaceKey (createReferenceHelper, disclosureUpdateReferenceHelper, exerciseInterfaceByKeyHelper)
 import Daml.Finance.Interface.Util.V3.InterfaceKey qualified as InterfaceKey (HasInterfaceKey(..))
+import Splice.Api.Token.HoldingV1 qualified as SpliceHolding
```

Comment:
Can we name the qualified import as follows

```suggestion
import Splice.Api.Token.HoldingV1 qualified as Api.HoldingV1
```

this is consistent with what we have been using in the Utilities repo (the Registry App)